addons/point_of_sale/static/src/app/screens/product_screen/order_summary/order_summary.js

```diff
@@ -52,13 +52,19 @@ export class OrderSummary extends Component {
         line.editPackLotLines(editedPackLotLines);
     }
 
-    clickLine(orderline) {
-        this.numberBuffer.reset();
-        if (orderline.isSelected()) {
-            this.pos.get_order().uiState.selected_orderline_uuid = null;
-        } else {
-            this.pos.selectOrderLine(this.currentOrder, orderline);
+    clickLine(e, orderline) {
+        if (e.detail === 2) {
+            clearTimeout(this.singleClick);
+            return;
         }
+        this.singleClick = setTimeout(() => {
+            this.numberBuffer.reset();
+            if (orderline.isSelected()) {
+                this.pos.get_order().uiState.selected_orderline_uuid = null;
+            } else {
+                this.pos.selectOrderLine(this.currentOrder, orderline);
+            }
+        }, 500);
```

Comment:
Since a click event can differentiate double from single click, maybe its better to handle `toggleSkipChange` and `selecting-deselecting` inside this `clickLine` event handler.